src/pages/index.js

```diff
@@ -331,7 +332,27 @@ const IndexPage = () => {
     </main>
   )
 }
+
 export default IndexPage;
 export const Head = () => (
-  <SEO/>
+  <SEO
+      title="Passion Corp - Go Passion Go World!"
+      description="Passion Corp is comprised of professional, compassionate and Event Planners and Assistants who love helping clients with special events."
+      keywords="Passion Corp, Passion Corp Indonesia"
+    >
+      <script type="application/ld+json">
+  {`
+    {
+      "@context": "https://schema.org",
+      "@type": "Organization",
+      "name": "Passion Corp",
+      "url": "https://passioncorp.id/",
+      "logo": "https://passioncorp.id/static/8bebf6dfc17e1b0c2e6fb7d1be3aa6d3/f9338/logo-white.webp",
```

Comment:
Sebaiknya jangan gunakan url ini, karena jika di clean direktori nya akan digenerate ulang dan url nya pasti berubah. Jika perlu public url image yang static, masukkan di folder `static` image nya.

Reply:
Kalo ga nanti biar saya aja yang update url logonya. Thank you.

Reply:
Halo mas happy,

Maaf nih saya belum bisa pegang laptop. Kalau bisa saya minta tolong sekalian mau liat buat pembelajaran kedepannya, saya masih belum mengerti gatsbyjs sepenuhnya.

Terima kasih.

Reply:
> Kalo ga nanti biar saya aja yang update url logonya. Thank you.

Saya sudah ganti dengan kode terbaru